exercises/practice/etl/.meta/config.json

```diff
@@ -1,5 +1,7 @@
 {
-  "authors": [],
+  "authors": [
+    "sturmer"
```

Comment:
This seems to be an unrelated change. I don't mind it _that_ much, but you could consider doing this in a separate PR.

Reply:
Will do. Definitely a case of “well one more tweak can’t hurt”.